Automatic AAB upload is erroring with a KeyError

## Observed behavior
Automated AAB upload is erroring with a KeyError when it is trying to look at the releases in a track.

## Errors and logs
```
2023-09-06T00:30:34.9919959Z make write-version
2023-09-06T00:30:34.9930451Z make[1]: Entering directory '/home/runner/work/kolibri/kolibri'
2023-09-06T00:30:34.9946087Z python3 scripts/version.py set_version_code
2023-09-06T00:30:36.1603990Z Traceback (most recent call last):
2023-09-06T00:30:36.1604868Z   File "/home/runner/work/kolibri/kolibri/scripts/version.py", line 106, in <module>
2023-09-06T00:30:36.1607911Z     write_build_number()
2023-09-06T00:30:36.1608671Z   File "/home/runner/work/kolibri/kolibri/scripts/version.py", line 71, in write_build_number
2023-09-06T00:30:36.1611068Z     f.write(str(_generate_build_number()))
2023-09-06T00:30:36.1611873Z   File "/home/runner/work/kolibri/kolibri/scripts/version.py", line 50, in _generate_build_number
2023-09-06T00:30:36.1614134Z     return get_latest_version_code() + 1
2023-09-06T00:30:36.1615003Z   File "/home/runner/work/kolibri/kolibri/scripts/play_store_api.py", line 64, in get_latest_version_code
2023-09-06T00:30:36.1617258Z     for release in track["releases"]:
2023-09-06T00:30:36.1619124Z KeyError: 'releases'
2023-09-06T00:30:36.1893370Z make[1]: *** [Makefile:128: .version-code] Error 1
2023-09-06T00:30:36.1894269Z make[1]: Leaving directory '/home/runner/work/kolibri/kolibri'
2023-09-06T00:30:36.1895201Z make: *** [Makefile:117: p4a_android_project] Error 2
2023-09-06T00:30:36.1911124Z ##[error]Process completed with exit code 2.
```

## Expected behavior
Should access the Play Store API without errors, as previously.

--- a/scripts/play_store_api.py
+++ b/scripts/play_store_api.py
@@ -61,7 +61,7 @@ def get_latest_version_code():
     )
     versionCode = 0
     for track in tracks:
-        for release in track["releases"]:
+        for release in track.get("releases", []):
             for vc in release.get("versionCodes", []):
                 versionCode = max(versionCode, int(vc))
     # Clean up after ourselves!

